src/python/pants/base/exception_sink.py

```diff
@@ -518,9 +518,6 @@ def _handle_signal_gracefully(cls, signum, signame, traceback_lines):
 
 # Setup global state such as signal handlers and sys.excepthook with probably-safe values at module
 # import time.
-# TODO: add testing for fatal errors at import-time, which may occur if there are errors in plugins.
-# Set the initial log location, for fatal errors during import time.
-ExceptionSink.reset_log_location(os.path.join(os.getcwd(), '.pants.d'))
```

Comment:
We should consider running this immediately after `init_workdir` in `pants_runner.py`, and having `init_workdir` return the workdir path to call it with.

Reply:
Move `reset_log_location` to `pants_runner.py`
```
    workdir_src = init_workdir(global_bootstrap_options)
    ExceptionSink.reset_log_location(workdir_src)
```